### PR TITLE
nim-falcon/0.0.1

### PR DESCRIPTION
**Pull request by pb-cdunn**
- drop osx (for now)
- depend on htslib/zlib at runtime

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pb-cdunn** [comment]
This used to work. Now it fails lint, so I'm trying to mimic **pbgcpp** but I get
```
ERROR: recipes/nim-falcon/meta.yaml:18: should_be_noarch_generic: The recipe should be build as `noarch`
```

**Comment by pb-cdunn** [comment]
https://bioconda.github.io/developer/_autosummary/bioconda_utils.lint.check_noarch.html

File: recipes/nim-falcon/meta.yaml [hunk 1]
```diff
@@ -9,22 +8,29 @@ package:
...
 
 build:
+  skip: True # [osx]
   number: 0
-  name: prebuilt
+
+requirements:
+  host:
+    - zlib
+    - htslib
```
Comment (pb-dseifert) [inline recipes/nim-falcon/meta.yaml]: don't need these at build-time?

Comment (pb-cdunn) [reply recipes/nim-falcon/meta.yaml]: Really? I don't see how the test could pass without them.

Comment (pb-dseifert) [reply recipes/nim-falcon/meta.yaml]: `host` is for build-time dependencies (and `test:` is a build-time phase)

**Comment by pb-cdunn** [comment]
This has been happening since the 2nd attempt today:
```
Linting Recipe(s)  Completed in 5s -- Failed to check out bioconda/bioconda-r
```

**Comment by pb-cdunn** [comment]
```
20:32:09 BIOCONDA INFO (OUT) + falconc -h
20:32:09 BIOCONDA INFO (OUT) could not load: libpcre.so(.3|.1|)
```
uh oh. That could be a problem with the prebuilts.

**Pull request merged**